core/client/src/test/java/alluxio/client/file/options/MountOptionsTest.java

```diff
@@ -40,6 +41,10 @@ public void fieldsTest() {
     Assert.assertTrue(options.isReadOnly());
     options.setReadOnly(false);
     Assert.assertFalse(options.isReadOnly());
+    options.setShared(true);
+    Assert.assertTrue(options.isShared());
+    options.setShared(false);
+    Assert.assertFalse(options.isShared());
```

Comment:
Agreed that it doesn't need to be in this PR, but it would be nicer if this was split into 1-line tests along the lines of `Assert.assertTrue(MountOptions.defaults().setShared(true).isShared())`